packages/fx-core/src/component/generator/utils.ts

```diff
@@ -135,19 +121,6 @@ export async function fetchTemplateZipUrl(
   return `${templateConfig.templateDownloadBaseURL}/${selectTemplateTag}/${name}.zip`;
```

Comment:
Seems there's a bug. It should be `${templateConfig.templateDownloadBaseURL}/${selectedTag}/${name}.zip`

Reply:
Thanks. Fixed.